rep-2000.rst

```diff
@@ -709,6 +709,121 @@ Package manager use for dependencies:
 - OpenEmbedded: opkg
 
 
+Build System Support:
+
+- ament_cmake
+- cmake
+- setuptools
+
+Galactic Geochelone (May 2021 - May 2022)
+----------------------------------
```

Comment:
Nit: expand the dashes to cover the whole previous line.

Reply:
f9c468e